app/views/songs/index.html.erb

```diff
@@ -39,7 +39,9 @@
       serverSide: true,
       responsive: true,
       lengthChange: false,
-      dom: '',
+      pageLength: 25,
```

Comment:
Oh yeah, forgot I was using Webrick. I'll leave the query stuff alone then. Pagination is already implemented though, that change was fairly minor. I needed it to make viewing songs more uh...manageable lol.